src/VisualStudio/Core/Def/Implementation/ProjectSystem/Legacy/AbstractLegacyProject.cs

```diff
@@ -172,7 +176,7 @@ protected sealed override void ValidateReferences()
         private void ValidateReferencesCore()
         {
             // can happen when project is unloaded and reloaded or in Venus (aspx) case
-            if (ProjectFilePath == null || TryGetBinOutputPath() == null || TryGetObjOutputPath() == null)
+            if (ProjectFilePath == null || BinOutputPath == null || ObjOutputPath == null)
```

Comment:
Why the rename here? I think we had the method so it was clear you might not get something back from these.